src/domain/reports/glass-admin/entities/GLASSDataMaintenanceItem.tsx

```diff
@@ -0,0 +1,77 @@
+import _ from "lodash";
+import { PaginatedObjects } from "../../../../types/d2-api";
+import { Id, NamedRef } from "../../../common/entities/Base";
+import { User } from "../../../common/entities/User";
+
+export type Status = "UPLOADED" | "VALIDATED" | "COMPLETED" | "DELETED";
+
+export type Module = "AMR" | "EGASP" | "AMRIndividual" | "EAR";
+
+export interface GLASSDataMaintenanceItem {
+    id: Id;
+    fileId: Id;
+    fileName: string;
+    module: Module;
+    orgUnit: string;
+    orgUnitName: string;
+    period: string;
+    status: Status;
+}
+
+export interface GLASSModule extends NamedRef {
+    name: Module;
+    userGroups: {
+        approveAccess: NamedRef[];
+    };
+}
+
+export interface GLASSMaintenancePaginatedObjects<T> extends PaginatedObjects<T> {
+    rowIds: string[];
+}
+
+export interface ATCItem {
+    currentVersion: boolean;
+    previousVersion: boolean;
+    uploadedDate: string;
+    version: string;
+    year: string;
+}
+
+export interface ATCItemIdentifier {
+    currentVersion: boolean;
+    version: string;
+    year: string;
+}
+
+export interface AMCRecalculation {
```

Comment:
This has to be like the following: 
```
export type AMCRecalculation = {
    date: string;
    recalculate: boolean;
    periods: string[];
    orgUnitsIds: Id[];
   loggerProgram: string;
};
```
Please, check that in all the code is `orgUnitsIds` instead of `orgUnitIds` and `date` instead of `currentDate` because this is how it was typed in GLASS app